.github/workflows/stale.yml

```diff
@@ -0,0 +1,26 @@
+name: Check stale issues and pull requests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
+          repo-token: ${{ secrets.PROJECT_BOARD_AUTOMATION }}
+          stale-issue-message: 'This issue is stale because it has been open for 30 days with no activity. It will be closed in 7 days unless the stale label is removed.'
+          stale-issue-label: stale
+          stale-pr-message: 'This pull request is stale because it has been open for 30 days with no activity. It will be closed in 7 days unless the stale label is removed.'
+          stale-pr-label: stale
+          days-before-stale: 30
+          days-before-close: 7
+          remove-stale-when-updated: true
+          exempt-issue-labels: 'keep'
```

Comment:
How about exempting [desirability labels](https://notes.ethereum.org/@solidity/SkMG8szAq#Desirability) instead? I think if the issue has one, we don't want it closed. The issues that we want people (and us) to be bugged about are those that have already been triaged but now need a decision - either to close them or to place them somewhere on our roadmap.

What do you think @ekpyron?

Reply:
Makes sense at least for now - if we eventually have too many ``nice to have`` issues going stale, we can still reconsider and change this :-).